src/mongoose_client_api/mongoose_client_api_sse.erl

```diff
@@ -21,6 +21,10 @@ init(_InitArgs, _LastEvtId, Req) ->
     ?LOG_DEBUG(#{what => client_api_sse_init, req => Req}),
     {cowboy_rest, Req1, State0} = mongoose_client_api:init(Req, []),
     {Authorization, Req2, State} = mongoose_client_api:is_authorized(Req1, State0),
+    % set 1h timeout to prevent from frequent client disconections
```

Comment:
I would make the option configurable with 1h being the default. Support infinity if possible as well.